batch/batch/batch.py

```diff
@@ -1160,8 +1164,9 @@ def is_successful(self):
     user = userdata['username']
     batches = await _get_batches_list(params, user)
     token = new_csrf_token()
-    context = {'batch_list': batches[::-1], 'token': token}
-
+    context = base_context(deploy_config, userdata, 'batch')
+    context['batch_list'] = batches[::-1]
+    context['token'] = token
     response = aiohttp_jinja2.render_template('batches.html',
```

Comment:
For consistency prefer @aiohttp_jinja2.template('batches.html')

Reply:
We can't here because we're setting the _csrf token.  I think this is the intended usage, see the "Using a function based web handler" example in the docs:

https://github.com/aio-libs/aiohttp-jinja2

Reply:
Ah interesting.